tests/test_ETC_initialization.py

```diff
@@ -128,30 +128,30 @@ def test_counts(telescope):
 
     #   For Version 1.0.0
```

Comment:
This comment is no longer true, maybe put the hash? Or we re-release and you update this to v1.0.1